src/lib/configuration.ts

```diff
@@ -188,6 +199,10 @@ export class Configuration {
       request.params = params;
     }
 
+    if (Configuration.accessKeys.network) {
+      request.headers['Network'] = Configuration.accessKeys.network;
```

Comment:
Assuming this is a HTTP request header, should this not be `x-network`? I would likely go further and narrow the scope of this header - for example, `x-blockchain-network` because "`network`" is a little too generic.

Reply:
That's a good point, actually, `Network` is a bit ambiguous. I like your idea better, but this is what the API expects. We could elevate this suggestion to the API implementation and consider it an option for the future :+1: